examples/dao/setup_sigma_dao.sql

```diff
@@ -11,6 +11,16 @@ CREATE TABLE IF NOT EXISTS sigma_daos (
   asset_id BIGINT -- token id
 );
 
+-- create a sigma_dao_proposal table if does not exit already.
+CREATE TABLE IF NOT EXISTS sigma_dao_proposals (
+  id SERIAL PRIMARY KEY, -- auto increment id
+  addr BYTEA,  -- account address
+  app BIGINT, -- app id
```

Comment:
this should be foreign key. Please add a constraint

Reply:
Done.